### PR TITLE
Implement pylint suggestions

### LINKED ISSUE #52: Dashboard - fix pylint `import error` issue
For the python files within the `pages` folder, there is a pylint import error regarding the files outside of the pages folder being imported. This needs to be fixed.

### PR DESCRIPTION
**Pull request by e-lemma**
Resolves #52.
I've reviewed the pylint scores for the dashboard files, and implemented the suggestions in order to raise all pylint scores to 8 or higher.
Thanks

### CODE REVIEW

File: dashboard/pages/artists.py [hunk 1]
```diff
@@ -2,30 +2,49 @@
...
 import streamlit as st
 
-from database import get_connection, get_popular_artists, get_all_artists, get_sales, get_sales_for_chosen_artists
-from charts import get_artist_album_sales_bar_chart, get_artist_track_sales_bar_chart, get_most_popular_artists_chart
+from database import (
+    get_connection,
+    get_popular_artists,
+    get_all_artists,
+    get_sales,
+    get_sales_for_chosen_artists,
```
Comment (Zhi-704) [inline dashboard/pages/artists.py:10]: Neat

File: dashboard/pages/subscriptions.py [hunk 2]
```diff
@@ -82,19 +87,29 @@ def show_subscriptions():
             st.write(f"You are now subscribed to the {tags} tag(s)")
             for tag in tags:
                 tag = tag.replace(" ", "-")
-                if f'arn:aws:sns:eu-west-2:129033205317:c11-bandcamp-{tag}' not in topic_arns:
+                if (
```
Comment (Zhi-704) [inline dashboard/pages/subscriptions.py:90]: Could you potentially just make this a global constant? Where it's something similar to 

BASE_URL = 'arn:aws:sns:eu-west-2:129033205317:c11-bandcamp-'

File: dashboard/pages/home.py [hunk 3]
```diff
@@ -13,7 +13,8 @@ def show_home():
 
     st.title("Home")
     st.header("Welcome to the Bandcamp analysis tracker!")
-    st.write("Here you'll find insights into Bandcamp sales and you can also subscribe to receive email notifications!")
+    st.write("""Here you'll find insights into Bandcamp sales
+             and you can also subscribe to receive email notifications!""")
```
Comment (Lasped13) [inline dashboard/pages/home.py:17]: Double check this. I'm pretty sure when you use triple quotes, the indents will be included in the new line.

Comment (e-lemma) [reply dashboard/pages/home.py:17]: 🤨 I've tested the dashboard after making this change and it looks the same

File: dashboard/pages/subscriptions.py (hunk 2 above)
Comment (e-lemma) [reply dashboard/pages/subscriptions.py:92]: Implemented now.

**Review by Zhi-704 — APPROVED** [review]

**Pull request merged**